packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -322,6 +326,8 @@ export class GarbageCollector implements IGarbageCollector {
     private readonly loggedUnreferencedEvents: Set<string> = new Set();
     // Queue for unreferenced events that should be logged the next time GC runs.
     private readonly pendingEventsQueue: IUnreferencedEvent[] = [];
+    // GC summary tree
```

Comment:
This comment doesn't provide any info. You should either remove it or add more info such as what this property is used for. For example, this stores the GC summary tree as per the last GC run.